<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BTform</class>
 <widget class="QWidget" name="BTform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>66</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBox_bt10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bt2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bt9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bt5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bt4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bt8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bt6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bt_all">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ALL</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bt1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bt7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_bt3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
